tests/adapters/switches/juniper_qfx_copper_test.py

```diff
@@ -31,7 +31,9 @@
 @ignore_deprecation_warnings
 def test_factory():
     lock = mock.Mock()
-    switch = juniper.qfx_copper_factory(SwitchDescriptor(hostname='hostname', model='juniper_qfx_copper', username='username', password='password', port=22), lock)
+    switch = juniper.qfx_copper_factory(
+        SwitchDescriptor(hostname='hostname', model='juniper_qfx_copper', username='username', password='password',
+                         port=22), lock)
 
     assert_that(switch, instance_of(FlowControlSwitch))
     assert_that(switch.wrapped_switch, instance_of(Juniper))
```

Comment:
Unrelated but weird, the test class is named JuniperTest ... the same than Juniper 
I hope it doesn't crash anything.